### PR TITLE
Handlebars helper

### PR DESCRIPTION
**Pull request by plesiecki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,17 +27,21 @@ Spring Boot Starter Handlebars will automatically register handlebars helpers ba
...
 Every other helper will register with its default name.
 
-More information about helpers can be found on
-[Handlebars.java](https://github.com/jknack/handlebars.java).
+More information about available helpers can be found on
+[Handlebars.java](https://github.com/jknack/handlebars.java#helpers).
+
+### Custom helpers
+
+To register helper use [@HandlebarsHelper](src/main/java/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelper.java) annotation.
```
Comment (platan) [inline README.md]: Maybe: To register helper -> To register a custom helper?

Comment (platan) [inline README.md]: We could link here to this resources. They document helper methods:
- https://github.com/jknack/handlebars.java/blob/master/handlebars/src/main/java/com/github/jknack/handlebars/HelperRegistry.java
- https://github.com/jknack/handlebars.java#using-a-helpersource

**Comment by platan** [comment]
Looks good :-)

File: README.md (hunk 1 above)
Comment (patrykpacewicz) [inline README.md]: IMHO There should be some usage example

**Pull request merged**